internal/gob/gob.go

```diff
@@ -0,0 +1,36 @@
+package gob
+
+import (
+	"encoding/gob"
+	"time"
+
+	dynamotypes "github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/jackc/pgx/v5/pgtype"
+	neosynctypes "github.com/nucleuscloud/neosync/internal/neosync-types"
+)
+
+// need to register all the types that are used in the connection data service
+// because we use interfaces for the types
+func RegisterGobTypes() {
+	gob.Register(map[string]any{})
+	gob.Register(map[string]any{})
+	gob.Register(pgtype.Numeric{}) // TODO fix this
```

Comment:
what is wrong with this?

Reply:
in the connection data stream and neosync data stream endpoint we shouldn't have any connection specific types. It should all be go types but it seems this one slipped through. Was going to fix in a follow up pr. Might be related to the big int issue I've been seeing

Reply:
sounds good